designer/apps/pages/migrations/0017_auto_20191018_1436.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2019-10-18 14:36
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailimages', '0019_delete_filter'),
+        ('pages', '0016_enterprisepagebranding_banner_background_color'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
since we already have data for branding on these program pages, will this migration be able to preserve that existing data?